extensions/amp-story/1.0/amp-story.js

```diff
@@ -21,6 +20,7 @@ import {VisibilityState_Enum} from '#core/constants/visibility-state';
 import {isRTL, removeElement} from '#core/dom';
 import {whenUpgradedToCustomElement} from '#core/dom/amp-element-helpers';
 import {escapeCssSelectorIdent} from '#core/dom/css-selectors';
+import * as Preact from '#core/dom/jsx';
```

Comment:
I know this was already here, but this feels really misleading given that a) this mirrors an existing, different convention in the repo and b) this actually has nothing to do with Preact and is exclusively JSX things. Suggest changing to `as JSX` or importing `{...}` directly (`jsx.js` only exports 4 identifiers anyway)

Reply:
For sure not changing this within this PR...but cc @alanorozco for thoughts since he created it.